README.md

```diff
@@ -1,3 +1,8 @@
+[![Open Source Love svg1](https://badges.frapsoft.com/os/v1/open-source.svg?v=103)](https://github.com/ellerbrock/open-source-badges/)
```

Comment:
I'm a bit hesitant about linking to images on someone else's server, It's probably ok but images can be used for analytics and other slightly nefarious activities.

Reply:
Yes that's a good point, probably better safe than sorry

Reply:
on reading up I think there is a notion of `marketing badges` and then badges that provide a bit more contextual information, I think those contextual ones might provide a bit more value, what do you think?

Reply:
Yes I think you're right, some dynamic badges with live information about the project would be more useful. I'll add the Github actions one, and also look into the Netlify one - and then perhaps look into others like coverage, etc.